dasharo-compatibility/display-ports-and-lcd-support.robot

```diff
@@ -77,6 +77,15 @@ DSP002.301 - External HDMI display in OS (Windows)
     Login To Windows
     Check HDMI Windows
 
+DSP002.203 - External HDMI display in OS (XCP-NG)
+    [Documentation]    Check whether an external HDMI display is visible in
+    ...    XCP-NG OS. An external HDMI display must be provided in
+    ...    the platform config.
+    Skip If    not ${EXTERNAL_HDMI_DISPLAY_SUPPORT}    DSP002.203 not supported
+    Skip If    not ${TESTS_IN_XCP_NG_SUPPORT}    DSP002.203 not supported
+    Skip If    "${ENV_ID_XCP_NG}" not in "${TESTED_LINUX_DISTROS}"    DSP002.203 not supported
+    Execute Manual Step    External HDMI Display    ${ENV_ID_XCP_NG}
```

Comment:
Why is it manual, if we do automated check in other Linux distros? Is tehre something specific to XCP-NG that would make the test different from e.g. Ubuntu or Fedora?